apps/coscrad-frontend/src/utils/generic-components/presenters/detail-views/resource-detail-full-view-presenter.tsx

```diff
@@ -40,13 +40,7 @@ export const ResourceDetailFullViewPresenter = ({
             </Grid>
             <Grid item xs={2} sm={2} md={8}>
                 {/* TODO: consider putting a standardized name property on the view models */}
-                <Typography gutterBottom variant="h6" fontWeight="bold" color="primary">
-                    {isString(name) || isNullOrUndefined(name) ? (
-                        name
-                    ) : (
-                        <MultilingualTextPresenter text={name} />
-                    )}
-                </Typography>
+                <ResourceDetailPresenterHeader id={id} type={type} name={name} variant="h3" />
```

Comment:
This is where good design pays off. If we didn't have our single source of truth (design standard) for resource detail full-views, we would have to use this header component once per resource. Good stuff!